README.md

```diff
@@ -668,6 +695,107 @@ Errors:
 
 - `ExpectationFalse`: If the value of the Error Register is not equal to the parameter.
 
+### `QueryPallet`
+
+Queries the existence of a particular pallet type.
+
+Operands:
+
+Kind: *Instruction*
+
+Errors:
+
+### `ExpectPallet`
+
+Ensure that a particular pallet with a particular version exists.
+
+Operands:
+
+Kind: *Instruction*
+
+Errors:
+
+### `ReportTransactStatus(QueryResponseInfo)`
+
+Send a `QueryResponse` message containing the value of the Transact Status Register to some destination.
+
+Operands:
+
+- `query_response_info`: The information needed for constructing and sending the  `QueryResponse` message.
+
+Kind: *Instruction*
+
+Errors: *Fallible*.
+
+### `ClearTransactStatus(QueryResponseInfo)`
+
+Set the Transact Status Register to its default, cleared, value.
+
+Operands: None
+
+Kind: *Instruction*
+
+Errors: *Infallible*.
+
+### `LockAsset(MultiAsset, MultiLocation)`
+
+  Lock the locally held asset and prevent further transfer or withdrawal.
+
+  This restriction may be removed by the `UnlockAsset` instruction being called with an
+  Origin of `unlocker` and a `target` equal to the current `Origin`.
```

Comment:
There is no mention of what `unlocker` or `target` is here. We should either avoid mentioning them, or better yet, rewrite the struct syntax above to instead be `LockAsset { asset: MultiAsset, unlocker: MultiLocation }`.